### DISCUSSION TITLE
How can I add Netlify Identity Widget

### DISCUSSION THREAD

**Discussion by sysadev** [Knogobert/ntn-boilerplate#11] [Q&A] [answered]
How can I add Netlify Identity Widget to the home page?

**Comment by Knogobert** [comment, marked as answer]
This Issue #8  is probably related to your question.

There's also some information in the Readme:
> ... Add the 'netlify-identity-widget' to nuxt.config.js under head, [Follow the advice here](https://answers.netlify.com/t/common-issue-netlify-cms-git-gateway-email-not-confirmed/10690)


But to be short to the point: 
Add this to your `nuxt.config.js` > `head` (on line 55):
```js
script: [{
  href: 'https://identity.netlify.com/v1/netlify-identity-widget.js'
}],
```

OR 


Add this `<script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>` before the `</head>` of an index.html
